src/Arrays/MPIStateArrays.jl

```diff
@@ -288,57 +290,50 @@ function knl_L2norm(::Val{Np}, Q, weights, elems) where {Np}
   energy
 end
 
-#=
-# {{{ L2 Error (for all dimensions)
-function AD.L2errornorm(runner::Runner{DeviceArray}, Qexact;
-                        host=false, Q = nothing, vgeo = nothing,
-                        time = nothing) where DeviceArray
-  host || error("Currently requires host configuration")
-  state = runner.state
-  config = runner.config
-  params = runner.params
-  cpubackend = DeviceArray == Array
-  if vgeo == nothing
-    vgeo = cpubackend ? config.vgeo : Array(config.vgeo)
-  end
-  if Q == nothing
-    Q = cpubackend ? state.Q : Array(state.Q)
-  end
-  if time == nothing
-    time = state.time[1]
+function euclideandist(A::MPIStateArray, B::MPIStateArray)
```

Comment:
Can we use camel case or underscores to make this and other function names more easily readable, e.g., `EuclideanDist`?

Reply:
I would suggest `euclidean_distance` over `EuclideanDist` since that would be more in line with the Julia style for functions (CamelCase tends to be used more for types/structs).

Reply:
I've updated a handful of other functions you might have been referring to. If there are others please let me know (or feel free to register an issue with a checklist or pop in a pull-request). For the most part I'm not wed to any names or naming conventions.

(I agree that `euclideandist` was a bad name, `euclidiandistance` would have been better at a minimum.)